apps/forklift/runtime.exs

```diff
@@ -129,3 +127,23 @@ if System.get_env("RUN_IN_KUBERNETES") do
       ]
     ]
 end
+
+config :telemetry_event,
+  metrics_port: System.get_env("METRICS_PORT") |> String.to_integer(),
+  metrics_options: [
+    [
+      metric_name: "events_handled.count",
+      tags: [:app, :author, :dataset_id, :event_type],
+      metric_type: "COUNTER"
+    ],
```

Comment:
Since these are duplicated across apps, should we look into having `telemetry_events` define these as functions so they're standardized?

Reply:
Note: this doesn't need to be a part of this pr